text/0000-rust-crate-ownership.md

```diff
@@ -0,0 +1,158 @@
+# Rust crate ownership policy
+
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, 2021-05-04)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+
+# Summary
+[summary]: #summary
+
+Have a more intentional policy around crates published by the Rust project, to be applied to existing and future crates published by us.
```

Comment:
```suggestion
Have a more intentional policy around crates published by the Rust project **on crates.io**, to be applied to existing and future crates published by us.
```
There seems to be a lot of confusion in the other comments about the fact that this only applies to crates published on `crates.io`. To reduce confusion, I would suggest that portion of the policy be very clearly and explicitly stated in the summary section. I's stated below in the motivation section, but it's sufficiently important to the policy that putting it in the summary section would be a good idea IMO.